app/controllers/api/projects.php

```diff
@@ -1693,6 +1698,7 @@
     ->desc('Delete platform')
     ->groups(['api', 'projects'])
     ->label('audits.event', 'platforms.delete')
+    ->label('audits.resource', 'project/{request.projectId}/platform/${request.platformId}')
```

Comment:
Maybe this should be `project/{request.projectId}` too? Please confirm with @christyjacob4.